### ISSUE TITLE
Ability to Add "Featured" datasets to new 4.7.1 Customizable Home Page

### ISSUE THREAD

**Issue by shlake** [IQSS/dataverse#4162]
Thank you for contributing an issue to the Dataverse Project! If this is a bug report, please let us know when the issue occurs, which page it occurs on, to whom it occurs, and which version of Dataverse you're using. If this is a feature request, please let us know what you'd like to see and give us some context - what kind of user is the feature intended for, and what inspired the request? No matter the issue, screenshots are always welcome.

**Comment by djbrooke** [comment]
@shlake - thanks for creating this issue following the discussion on the call earlier today!

**Comment by edzale** [comment]
This would really be nice to be able to feature datasets: as the number of datasets is growing in a (D)ataverse or (d)ataverse, we really feel the need to somehow editorialize its content. Thank you for opening this issue.

**Comment by TaniaSchlatter** [comment]
@slake and @edzale, would you want the featured datasets shown to be defined completely manually or are there some criteria for what you want to display?

**Comment by shlake** [comment]
OK, maybe I created this with a bad title - Reading my initial description I say "like those that show up at the top of the "normal" dataverse home page."

These would be "featured **dataverses**", not datasets (as the title says).

Not sure if @edzale meant "datasets" or "dataverses".

For me, if I have made a **dataverse** featured, then I have already thought it was important, so that's what I would want to see on the Customizable page - a call to show "Featured" dataverses.

Hmm.... but featured dataverses from root dataverse, or any featured dataverse??? That wouldn't work as users could also "feature" sub-dataverses in their own dataverse.....

Maybe admin being able to pick what shows up on the Customized home page for Featured dataverse, would be best.

Maybe a similar thing for "featured datasets", which would be a different issue. Are there two issues here?

- show featured dataverses on Customized Home page (admin can select which to show)
- show featured datasets on Customized Home page (admin can select which to show)

**Comment by edzale** [comment]
@shlake, I meant "datasets". In my case, I would like to be able to show datasets that seem important to us on the home page.
@TaniaSchlatter, it could be a mix. For example, datasets that are the most downloaded could be automatically feautured, but we need to also to proceed manually.

**Comment by slake** [comment]
> @slake and @edzale, would you want the featured datasets shown to be defined completely manually or are there some criteria for what you want to display?

Hi , wrong login mentioned.

**Comment by pdurbin** [comment]
@shlake @edzale any thoughts on what we should do with this issue? Perhaps we could close it and discuss on the mailing list or https://chat.dataverse.org or during a community call? Or create a Google doc to sketch out some ideas? Issues aren't the best place for discussion, I'm afraid. Thanks.

**Comment by edzale** [comment]
@pdurbin, yes sketching out some ideas in a google doc seems a good idea to me.

**Comment by cmbz** [comment]
2024/09/09: Keeping open. With SPA work in progress, this might be easier to achieve.
